src/Authentication/Actions/Email2FA.php

```diff
@@ -132,7 +132,7 @@ public function verify(IncomingRequest $request)
     public function createIdentity(User $user): string
     {
         /** @var UserIdentityModel $identityModel */
-        $identityModel = model(UserIdentityModel::class);
+        $identityModel = model('UserIdentityModel');
```

Comment:
At this time, we have not received any requests that this change is necessary. 
So I think it is fine to leave `::class` as it is.

Also, since we provide the config to replace the User Provider, it does not seem necessary for devs to replace other model from a module.

About Config files, #761 was created.

Reply:
So we need to lock the **codeigniter/phpstan-codeigniter** to the specific version.
 
https://github.com/codeigniter4/shield/blob/fd2dff93921c3789180cc17b9fb79b9d63aafc9b/composer.json#L32

Reply:
We can ignore the error.
See https://github.com/codeigniter4/CodeIgniter4/blob/52dbf70837526e9cdbd948248182006d17044f92/phpstan.neon.dist#L41-L42